client/v1/request.js

```diff
@@ -317,8 +317,10 @@ Request.prototype.errorMiddleware = function (response) {
         throw new Exceptions.ActionSpamError(json);
     if (json.message == 'checkpoint_required')
         throw new Exceptions.CheckpointError(json, this.session);
-    if (json.message == 'login_required')
-        throw new Exceptions.AuthenticationError("Login required to process this request");
+    if (json.message == 'checkpoint_required')
```

Comment:
Is this correct? the login_required exception has been removed and I see the checkpoint_required duplicated